src/main/java/org/cactoos/text/LowerText.java

```diff
@@ -59,13 +50,7 @@ public LowerText(final Text text) {
      * @param lang Locale
      */
     public LowerText(final Text text, final Locale lang) {
-        this.origin = text;
-        this.locale = lang;
-    }
-
-    @Override
-    public String asString() throws Exception {
-        return this.origin.asString().toLowerCase(this.locale);
+        super((Scalar<String>) () -> text.asString().toLowerCase(lang));
```

Comment:
@krzyk We should not have this transformation to lower case here in the constructor, just like above.